Modules/Relationships/src/Relationships.Application/ApplicationErrors.cs

```diff
@@ -1,3 +1,30 @@
+using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+
 namespace Backbone.Modules.Relationships.Application;
 
-public static class ApplicationErrors;
+public static class ApplicationErrors
+{
+    public static class Relationship
+    {
+        public static ApplicationError RelationshipToTargetAlreadyExists(string targetIdentity = "")
+        {
+            var targetIdentityString = string.IsNullOrEmpty(targetIdentity) ? "the target identity" : targetIdentity;
+
+            return new ApplicationError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists", $"A relationship to {targetIdentityString} already exists.");
+        }
+
+        public static ApplicationError CannotSendRelationshipRequestToYourself()
+        {
+            return new ApplicationError("error.platform.validation.relationshipRequest.cannotSendRelationshipRequestToYourself", "The template you provided is your own. You cannot send a relationship request to yourself.");
+        }
```

Comment:
Why did you add these errors?

Reply:
I've used them at some point and then I refactored the code. I forgot to remove these unfortunately.

Changes made.